helm-chart/values.yaml

```diff
@@ -59,32 +60,26 @@ serviceMonitor:
 # Prometheus annotations to configure scraping metrics from the connector
 prometheus:
```

Comment:
hmm I think there is some discrepancy here. Originally this section was about Promscale exposing it's metrics to Prometheus. Not about the remote_write interface. So prometheus.enabled actually just controls the /metrics endpoint and /not/  the web interface of remote_write (we actually have no way to turn that off ATM). It seems like putting the port for remote write here is wrong unless we change the semantics of the section (but then prometheus.enabled doesn't work).

Let me make a suggestion: let's add a metric top-level section and put the port there. That makes `tracing` and `metrics` have symmetry too.

Reply:
I think I don't get something in this comment.

> So prometheus.enabled actually just controls the /metrics endpoint and /not/ the web interface of remote_write

`prometheus.enabled` controls if annotations are set on pod or not. IMHO this switch should be removed completely and we should just rely on `prometheus.annotations` section.

> It seems like putting the port for remote write

The port is the same for metrics exposition and remote_write.

>  let's add a metric top-level section and put the port there. 

AFAIK that is something we want to avoid as `metrics` can mean prometheus metrics as well as OT metrics.

Reply:
> prometheus.enabled controls if annotations are set on pod or not. IMHO this switch should be removed completely and we should just rely on prometheus.annotations section.

Yup because originally we used annotation-based service discovery in Prometheus and turning off the annotations meant Promscale wasn't scraped by Prometheus. I agree this is silly. Let's remove prometheus.enabled in this PR and just rely on `prometheus.annotations` section as you suggest.

> AFAIK that is something we want to avoid as metrics can mean prometheus metrics as well as OT metrics.

Good point!

Ok I think I am fine with this change in meaning as long as we remove `prometheus.enabled` (since otherwise I think we give the wrong impression that remote_write can be disabled).